README.md

```diff
@@ -0,0 +1,38 @@
+# Operator for OSBuilder
+
+This repo has the operator which provides a K8S API for building [OSBuild](https://www.osbuild.org/) images.
+
+## Installation
+
+### Requirements
+
+- `osbuild >= 26`
+- `go >= 1.16`
+
+## Getting started
+
+### Deployment
+
+- Build and push image:
+
+    `IMG=<image repository and tag> make docker-build docker-push`
+
+    for example: `IMG=quay.io/project-flotta/osbuild-operator:latest make docker-build docker-push`
+- Deploy the operator:
+
+    `IMG=<image repository and tag> make deploy`
+
+    for example: `IMG=quay.io/project-flotta/osbuild-operator:latest make deploy`
+
+In the `osbuild-operator-system` namespace an operator should be running:
+```
+→ oc get pods -n osbuild-operator-system
```

Comment:
in next task, let's replace the namespace with simply `osbuild`

Reply:
done in #17